package/forms.py

```diff
@@ -11,6 +12,7 @@
 from django.forms.widgets import Textarea, TextInput
 from django.template.defaultfilters import slugify
 from floppyforms.__future__ import ModelForm
+from django.conf import settings
```

Comment:
Also use "Optimize imports" (`Ctrl` + `Shift` + `A` and type `optimize imports`, or just `Ctrl` + `Alt` + `O`) feature of PyCharm, to sort imports